src/Tribe/Repositories/Event.php

```diff
@@ -1471,23 +1471,28 @@ public function handle_order_by( $order_by ) {
 		$loop  = 0;
 
 		foreach ( $check_orderby as $key => $value ) {
-			$loop++;
 			$order_by      = is_numeric( $key ) ? $value : $key;
-			$order         = is_numeric( $key ) ? 'ASC' : $value;
 			$default_order = Arr::get_in_any( [ $this->query_args, $this->default_args ], 'order', 'ASC' );
+			$order         = is_numeric( $key ) ? $default_order : $value;
+
+			// Let the first applied ORDER BY clause override the existing ones, then stack the ORDER BY clauses.
+			$override = $loop === 0;
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: fb1bda9b5faff3f6e42ac5240d3049e1</sub>